proseco/guide.py

```diff
@@ -70,6 +106,8 @@ def get_guide_catalog(obsid=0, **kwargs):
 
     # Do a first cut of the stars to get a set of reasonable candidates
     guides.cand_guides = guides.get_initial_guide_candidates()
+    if just_initial_cands:
+        return guides.cand_guides
```

Comment:
There's more work to be done to have a clean function to get the initial candidates without this early-exit logic.